tests/components/sentry/test_config_flow.py

```diff
@@ -34,23 +33,51 @@ async def test_form(hass):
         "dsn": "http://public@sentry.local/1",
     }
     await hass.async_block_till_done()
+
     assert len(mock_setup.mock_calls) == 1
     assert len(mock_setup_entry.mock_calls) == 1
 
 
-async def test_form_bad_dsn(hass):
+async def test_integration_already_exists(hass):
+    """Test we only allow a single config flow."""
+    MockConfigEntry(domain=DOMAIN).add_to_hass(hass)
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+    assert result["type"] == "abort"
+    assert result["reason"] == "already_configured"
```

Comment:
Should we change reason to single instance allowed?

Reply:
Agree, this was also a review comment on an earlier review of a closed PR on the Todo list. I like to keep this PR aimed towards the removal of YAML and getting full coverage on the current situation. Mainly to provide a base to work on.